.circleci/config.yml

```diff
@@ -200,10 +199,17 @@ jobs:
       - attach_workspace:
           at: ~/dash
       - run:
-          name: 🧪 Run Integration Tests
+          name: ️️🏗️  Install packages
           command: |
             . venv/bin/activate && cd packages && ls -la
-            find . -name "*.gz" | xargs pip install --no-cache-dir --ignore-installed && pip list | grep dash && cd ..
+            find . -name "*.gz" | xargs pip install --no-cache-dir --ignore-installed && cd ..
+            sed -i '/dash/d' requires-install.txt
+            pip install --no-cache-dir --ignore-installed .
```

Comment:
FYI @byronz - the issue I was having in this PR is related to the original one that led to the `sed` call during `build-core`: sometimes (seemingly at random) the `dash-renderer` we prepared in `packages` was being overwritten by a published one from PyPI - so I used essentially the same solution, but extended to `dash-renderer` by using the pattern `/dash/d` rather than `/dash-/d`.

Also I skipped creating a tarball for dash itself because (a) we already have everything we need here, dash has no build artifacts, and (b) that made it easier to modify its requirements.